molecule/create-destroy/molecule.yml

```diff
@@ -0,0 +1,31 @@
+---
+dependency:
+  name: galaxy
+  options:
+    force: false  # don't fetch every time
+    role-file: molecule/_common/requirements.yml
+    requirements-file: molecule/_common/requirements.yml
+driver:
+  name: docker
+platforms:
+  - name: postgresql-objects-scenario-create-destroy
+    image: postgres:latest
+    command: postgres
+    dockerfile: ../_common/Dockerfile.j2
+    pre_build_image: false
+provisioner:
+  name: ansible
+verifier:
+  name: ansible
+scenario:
+  test_sequence:
+    - dependency
+    - lint
+    - cleanup
+    - destroy
+    - syntax
+    - create
+    - prepare
+    - converge
+    - cleanup
+    - destroy
```

Comment:
What does all this do? Just for my understanding

Reply:
Yeah I should add a comment. Basically, `molecule test` runs a [default sequence of actions](https://molecule.readthedocs.io/en/latest/configuration.html#scenario), and this overrides that default, mainly to remove `idempotence`, since the `create-destroy` scenario is intentionally not idempotent.